cmake/catkin_package.cmake

```diff
@@ -432,6 +432,31 @@ function(_catkin_package)
   # package cmake dir is the folder where the generated pkgConfig.cmake is located
   set(PKG_CMAKE_DIR "\${${PROJECT_NAME}_DIR}")
 
+  # strip SYSROOT prefixes for each element in LIST
+  macro(catkin_strip_sysroot)
+    cmake_parse_arguments(_fn "UNIQUE" "LIST" "" ${ARGN})
```

Comment:
Instead of a `LIST` argument the function should have one positional argument like `function(catkin_strip_root_path var)`. Also `var` is the common name for this use. Instead of `_fn` the prefix should be named `ARG` (for consistency with other code).